docs/quickstart.md

```diff
@@ -3,13 +3,22 @@ import TabItem from '@theme/TabItem';
 import IframeVideo from '@site/src/components/IframeVideo/index';
 import LinkButton from '@site/src/components/LinkButton/index';
 import SdkInstallation from './\_sdk-installation.mdx';
+import ThemedImage from '@theme/ThemedImage';
 
 # Blind App Quickstart
 
 ## What is a blind app?
 
 A blind app runs blind computation on Nillion using one or more privacy-preserving [Nada](/nada-lang) programs. Nada programs compute on secret integers without ever seeing the underlying input values, making them ideal for sensitive data operations.
 
+<ThemedImage
```

Comment:
Love the image addition here! Maybe it'd be worth adding a note or something right below that quickly explains that everything you see in this image is the output of create-nillion-app to give people context that this is what they will build in the Blind app Quickstart